src/ocean/core/BitArray.d

```diff
@@ -189,13 +191,8 @@ struct BitArray
     }
     body
     {
-        size_t mDim=len/32;
-        ptr[0..mDim] = rhs.ptr[0..mDim];
-        int rest=cast(int)(len & cast(size_t)31u);
```

Comment:
> This patch uses `memcpy()` to copy all the content from another BitArray instead of applying multiple bit operations.

Yes, but then you could just remove this multiple bit operations and just write

```
    BitArray opSliceAssign(BitArray rhs)                                                                               
    in                                                                                                                 
    {                                                                                                                  
        assert(rhs.len == len);                                                                                        
    }                                                                                                                  
    body                                                                                                               
    {                                                                                                                  
        size_t mDim=len/32;                                                                                            
        ptr[0..mDim] = rhs.ptr[0..mDim];                                                                               
        return *this;
}
```

because druntime will actually perform `memcpy` on the array while copying.